data/phones/arm_w_phonetic.phones

```diff
@@ -1,71 +1,79 @@
 # Based on https://en.wikipedia.org/wiki/Armenian_language#Phonology
+# And based on the pronunciation script from Wiktionary: https://en.wiktionary.org/wiki/Module:hy-pronunciation
+#
 # Vowels.
 ɑ
 ɛ
 ə
 i
 ɔ
-o  # <ո> is really [ɔ].
-a  # <ա> is really [ɑ].
 u
+#
+# Older entries might contain [o] or [e]. These have been fixed. You shouldn't find these phones anymore. 
+#
+# Long vowels: A sequence of identical heterosyllabic vowels is phonetically pronounced as one long vowel. Armenian does not have phonemic vowel length. But these are very rare. The only attested case on Wiktionary is the following
+ɛː
+# The other types of long vowels are not found on Wiktionary as of Jan 2021. This is largely because long vowels are mostly in loanwords and borrowings, which Wiktionary underreports. At some point in the future, the Armenian users might add some entries which contain these sequences.
+# ɑː əː iː ɔː uː
+#
 # Consonants.
 m
-mː
 n
-nː
 ŋ
 pʰ
-pʰː
-p
-pː
 b
-bː
 tʰ
-tʰː
-t
-tː
 d
-dː
 kʰ
-kʰː
-k
-kː
 ɡ
-ɡː
 t͡sʰ
-t͡sʰː
-t͡s  # No corresponding geminate?
-d͡z  # No corresponding geminate?
+d͡z
 t͡ʃʰ
-t͡ʃ
-d͡ʒ  # No corresponding geminate?
+d͡ʒ
 f
-fː
 v
-vː
 s
-sː
 z
-zː
 ʃ
+ʒ
+χ  
+ʁ
+h
+l
+j
+ɾ
+#
+# Past errors: some instances of [χ] were incorrectly transcribed as [x]. These were fixed.
+# Past errors: some affricates were missing a tie-bar. These were fixed.
+#  * <ց> [tʃʰ] 
+#  * <չ> [tsʰ]
+#
+# Long consonants: A sequence of identical consonants are phonetically pronounced as long consonant. Armenian does not have phonemic consonant length or geminates. This is just an accidental gap. The above long consonants can exist in Armenian. At some point in the future, the Armenian users might add some entries which contain these sequences
```

Comment:
Thanks for these detailed comments: should prevent our future selves from making these mistakes in the future.

Minor nit: could you add trailing sentential punctuation to these comments? You might also want to hard-wrap them at 80 chars. There was an example above of this.

Reply:
Sure. I tidied up. You should see the commit message "cleaned up armenian phones (with more tidying up)"

Reply:
Rats, I placed it on the wrong spot on the changelog -- I fixed it.